lib/rules/jsx-no-bind.js

```diff
@@ -5,13 +5,25 @@
  */
 'use strict';
 
+var Components = require('../util/Components');
 var propName = require('jsx-ast-utils/propName');
 
 // -----------------------------------------------------------------------------
 // Rule Definition
 // -----------------------------------------------------------------------------
 
-module.exports = function(context) {
+function renderReturnsJSX(node, utils) {
```

Comment:
Is there a reason that `isReturningJSX` isn't already checking for a return statement?